### ISSUE TITLE
flip: support dice notation

### ISSUE THREAD

**Issue by newmania** [keybase/client#16924]
This is maybe the lowest-priority request in the history of Keybase, but it would be nice to add [dice notation](https://en.wikipedia.org/wiki/Dice_notation) support to /flip for people who play D&D or other dice-based games. E.g.
`/flip 2d6+4`

**Comment by ponsfrilus** [comment]
Agreed, `/flip dice` and more advanced option for specifying the number of dices to throw and dices' sides would be fantastic !
